### PR TITLE
{bio}[foss/2021a] rnaQUAST v2.2.2 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by wdlingit**
added rnaQUAST-2.2.2-foss-2021a-Python-3.9.6.eb

requires:

- [x] #15375

### CODE REVIEW

**Comment by wdlingit** [comment]
this one is depending on  #15375

**Comment by migueldiascosta** [comment]
@wdlingit not sure if you meant to use `foss/2021b` with those versions of dependencies (except `GMAP-GSNAP`, which should then be the one mentioned in the comment above) or `foss/2021a`, in which case the versions of `Python`, `matplotlib` and `BLAST+` should be different

also note that in either case, the Python `versionsuffix` shouldn't be necessary anymore

**Comment by wdlingit** [comment]
@migueldiascosta Thank you for pointing out this. I am confusing about the `versionsuffix`. Sorry to all that this should be foss/2021b. I mistakenly built the easyconfig of wrong version while I was updating this one.

**Comment by migueldiascosta** [comment]
@wdlingit no worries, it can be confusing. the Python `versionsuffix` is no longer necessary when using the default version of Python for that toolchain.

also just noticed that upstream seems to have tagged `v.2.2.2` (note the extra dot after `v`, so this needs to be changed in `sources`) and the checksum is different?

**Comment by wdlingit** [comment]
@migueldiascosta I have updated the file name and its contents ("rnaQUAST with corrected foss ersion" (typo again)). This one should be the one I used to build my programs.

File: easybuild/easyconfigs/r/rnaQUAST/rnaQUAST-2.2.2-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+easyblock = 'Tarball'
+
+name = 'rnaQUAST'
+version = '2.2.2'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/rnaQUAST/rnaQUAST-2.2.2-foss-2021b.eb:5]:
```suggestion

```

Comment (wdlingit) [reply easybuild/easyconfigs/r/rnaQUAST/rnaQUAST-2.2.2-foss-2021b.eb]: @migueldiascosta Done and thank you!

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @wdlingit!

**Pull request merged**